src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -546,66 +580,65 @@ string Visitor::visitTypesafe(Parser::PnameNsContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-SparqlQleverVisitor::PrefixMap SparqlQleverVisitor::visitTypesafe(
+SparqlQleverVisitor::PrefixMap SparqlQleverVisitor::visit(
     SparqlAutomaticParser::PrologueContext* ctx) {
   if (!ctx->baseDecl().empty()) {
     reportError(ctx->baseDecl(0), "BaseDecl is not supported.");
   }
   for (const auto& prefix : ctx->prefixDecl()) {
-    visitTypesafe(prefix);
+    visit(prefix);
   }
   // TODO: we return a part of our internal state here. This will go away when
   //  queries can be parsed completely with ANTLR.
   return _prefixMap;
```

Comment:
Can this TODO and the return be removed by now, and the function made `void` as a return type?

Reply:
Hmm I thought that we also used that for testing but that does not seem to be the case.